torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -445,6 +445,33 @@ void PyTorchModelLoader::loadMaxPool2d(const torch::jit::Node *ptNode) {
   addGlowNodeValue(outputs[0], output);
 }
 
+void PyTorchModelLoader::loadReshape(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  assert(inputs.size() == 2);
+  assert(outputs.size() == 1);
+
+  // Indexes of aten::reshape inputs.
+  struct Inputs {
+	enum {
```

Comment:
formatting looks weird here?

Reply:
Hm, I guess we're not running the clang-format CI check in here...?

Reply:
Oh damn good catch, lets fix that

Reply:
https://github.com/pytorch/glow/pull/3304